src/Navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 3em;
+  color: white;
+  background-color: #5757ff;
+}
+
+.col-1 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+.btn {
+  background-color: transparent;
+  border: none;
+}
+
+.fa-chevron-down, .fa-plus {
+    margin-left: 0.5em;
+}
+
+.col-2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    padding: 1em 0;
+}
+
+.input {
+    background-color: #7b7bfd;
+    padding: 0.5em 1em;
+}
+
+.fa-search {
```

Comment:
Create a new class for spacing, by doing that where ever we will use `fa-search` class so we will have the margin-right by default as well which is not a good practice.